Frontend/MoptPaymentPayone/Subscribers/Payment.php

```diff
@@ -307,6 +314,12 @@ public function onShippingPaymentAction(\Enlight_Hook_HookArgs $arguments)
         $groupedPaymentMeans = $moptPayoneMain->getPaymentHelper()->groupCreditcards($subject->View()->sPayments);
         if ($groupedPaymentMeans) {
             $subject->View()->sPayments = $groupedPaymentMeans;
+            $this->container->get('events')->notify(
+                'MoptPaymentPayone_Subscriber_Payment_onShippingPaymentAction_paymentsGrouped',
+                [
+                    'groupedPaymentMeans' => $paymentMeansWithGroupedCreditcard
```

Comment:
See https://scrutinizer-ci.com/g/PAYONE-GmbH/shopware-5/inspections/40009b0a-d3fc-4d9f-81be-f73d5773b587/issues/files/Frontend/MoptPaymentPayone/Subscribers/Payment.php?status=new&orderField=path&order=asc&honorSelectedPaths=0

Should be `$groupedPaymentMeans`, right?